.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        node-version: ["14", "16"]
+        node-version: ["16"]
```

Comment:
@phated Should we consider adding 17 here?

Reply:
Never consider an odd node version. Odd node versions are pre-release versions and can break at any time with no semantic meaning.